.github/workflows/release.yaml

```diff
@@ -11,29 +11,27 @@ on:
 env:
   # bleh
   MAINNET: ${{ github.ref == 'refs/heads/mainnet/main' || (startsWith(github.ref,'refs/tags/v') && ( contains(github.ref,'.0.') || contains(github.ref,'.2.') || contains(github.ref,'.4.') || contains(github.ref,'.6.') || contains(github.ref,'.8.') )) }}
```

Comment:
You know, on `mainnet/main` I hastily flipped the `BUILD_MAINNET` logic in the `Makefile`.  It now defaults to mainnet unless `MAINNET=false`.  We should probably keep this in sync with that at some point.  Not necessary now, just thought of it.

Reply:
oh, i forgot that. this should turn into make target, otherwise won't peek MAINNET
```yaml
        run: |-
          docker run \
          --priviledged \
          -e MAINNET=${MAINNET} \
          -e DOCKER_USERNAME=${{secrets.DOCKERHUB_USERNAME}} \
          -e DOCKER_PASSWORD=${{secrets.DOCKERHUB_PASSWORD}} \
          -e GITHUB_TOKEN=${{ secrets.GORELEASER_ACCESS_TOKEN }} \
          -v /var/run/docker.sock:/var/run/docker.sock \
          -v `pwd`:/go/src/github.com/ovrclk/akash \
          goreng/golang-cross:${GOLANG_CROSS_VERSION} \
          release --rm-dist
```

Reply:
wait, actually, are we using the makefile with golang-cross?

what is important is that the build tag `mainnet` is set when compiling for the binary release.